spec/docs/dqt_records_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+require "swagger_helper"
+
+describe "API", type: :request, swagger_doc: "v1/api_spec.json" do
+  let(:token) { NpqRegistrationApiToken.create_with_random_token! }
+  let(:bearer_token) { "Bearer #{token}" }
+  let(:Authorization) { bearer_token }
+  let(:teacher_reference_number) { "1234567" }
+
+  before do
+    stub_request(:get, "http://api/qualified-teachers/qualified-teaching-status?ni&trn=1234567")
```

Comment:
I get what `trn` is, what is `ni` in there?

Reply:
`ni` will be short for national insurance number. this is generated by the dqt client stuff that was recently added.